packages/destination-actions/src/destinations/amplitude/__tests__/amplitude.test.ts

```diff
@@ -395,6 +395,59 @@ describe('Amplitude', () => {
       `)
     })
 
+    it('supports session_id from `integrations.Actions Amplitude.session_id` in number format', async () => {
+      const event = createTestEvent({
+        timestamp,
+        event: 'Test Event',
+        anonymousId: 'julio',
+        integrations: {
+          // @ts-expect-error integrations should accept complext objects;
```

Comment:
```suggestion
          // @ts-expect-error integrations should accept complex objects;
```

Is this comment required? Should we disable a linter rule instead?

Reply:
@marinhero Even if we disable a rule then we will be doing it for a single line temporarily. And to disable that rule we will need to add a comment. So, I think it will not make any difference.

We have corrected the spelling mistake in the comment as per your suggestion.